### ISSUE TITLE
Setting tabIndex

### ISSUE THREAD

**Issue by S0PEX** [arthurdenner/react-semantic-ui-datepickers#509]
# ❔ Question

Hey,

I am really enjoying your date picker but I can't get the tab order working in my form when using the date picker.
Currently I am using react-hook-form and added the tab index attribute to the date-picker but it isn't picked up by the browser. Do you have any solution to this ?
```tsx
        <Controller
          render={({ onChange, onBlur }) => (
            <SemanticDatepicker
              label="Datum"
              onBlur={onBlur}
              onChange={(_, { value }) => onChange(value)}
              locale="de-DE"
              placeholder="DD-MM-YYYY"
              format="DD-MM-YYYY"
              value={defaults?.start ? new Date(defaults?.start) : null}
              error={errors.startDate?.message}
              //@ts-ignore
              tabIndex='3'
            />)}
          name="startDate"
          defaultValue={defaults?.start ? new Date(defaults?.start) : null}
          control={control}
          disabled={readonly}
        //ref={register()}
        />
```

Regards Artur

**Comment by arthurdenner** [comment]
Hey @S0PEX! 👋🏽 

Sorry for the wait to reply and thank you for reporting this.

I looked into this issue today and managed to fix it by adding `tabIndex` to the "whitelist" of props forwarded to React Semantic UI's Input.

I want to get rid of this "whitelist" and just forward everything but it'll require more work, so I'll skip it for now and just fix this particular issue.

I've published `2.12.1-beta.1` to npm. Can you please try it and see if it works for you?

**Comment by S0PEX** [comment]
@arthurdenner Sorry for the late reply. I currently have my exam period at the university so I am currently a bit busy . I will see that I test it on Monday when I am back in the office.

Thanks for responding and the fix.

**Comment by arthurdenner** [comment]
Solved in #511. Please let me know if you still find issues.

**Comment by S0PEX** [comment]
Hey, I have just tested it. Works fine thanks for the efford.
